oblib/tests/test_data_model.py

```diff
@@ -865,10 +869,27 @@ def test_input_ids(self):
 
     def test_json_fields_are_strings(self):
         # Issue #77 - all json fields should be strings other than None which should convert
-        # a JSON null literal.
-        # e.g. numbers should be "100" not 100
-        # booleans should be "true" not true
-
+        # a JSON null literal.  e.g. numbers should be "100" not 100
+        # Issue #142 -- and booleans should convert to a JSON true/false literal.
+
+        """
+        "key": false - correct
+        "key": "false" - incorrect
+        "key": "False" - incorrect
+        "key": "0" - incorrect
+        "key": 0 - incorrect
+    
+        "key": true - correct
+        "key": "true" - incorrect
+        "key": "True" - incorrect
+        "key": "1" - incorrect
+        "key": 1 - incorrect
+
+        "key": null - correct
+        "key": "null" - incorrect
+        "key": "Null" - incorrect
+        "key": "None" - incorrect
+        """
```

Comment:
Add to comment:

NOTE: for the first release incorrect fields are processed on input but not on output.  This decision may be revisited in a future release.

I think if this is not added it may confuse someone.

Reply:
Added comment.